crates/storage/db-rocks/Cargo.toml

```
@@ -0,0 +1,75 @@
+[package]
+name = "reth-db-rocks"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[dependencies]
+
+# reth dependencies
+reth-primitives.workspace = true
+reth-db-api.workspace = true
+reth-db.workspace = true
+reth-codecs.workspace = true
+reth-storage-api.workspace = true
+# reth-trie = { workspace = true }
+reth-trie = { workspace = true, features = ["test-utils"] }
+reth-trie-db = { workspace = true }
+reth-trie-common = { workspace = true }
+alloy-primitives = { workspace = true }
+reth-primitives-traits = { workspace = true }
+reth-execution-errors = { workspace = true }
+alloy-rlp = { workspace = true }
+
+# rocksdb
+rocksdb = { version = "0.22.0" }
+serde = { version = "1.0.218", features = ["derive"] }
```

```suggestion
rocksdb = { version = "0.22.0" }
serde = { version = "1.0.218", features = ["derive"] }
```
It good to define it as workspace dependency

---

DOne